guides/doc-Upgrading_and_Updating/master.adoc

```diff
@@ -25,38 +25,44 @@ include::common/modules/proc_providing-feedback-on-red-hat-documentation.adoc[le
 endif::[]
 
 // Upgrading Satellite Server overview
-include::topics/upgrading_overview.adoc[leveloffset=+1]
+include::topics/con_upgrading-overview.adoc[leveloffset=+1]
```

Comment:
We can totally opt-out of `topics` and go for `assembly` or directly the `master`.

Reply:
Hey @mjivraja I'm moving the entire guide to an assembly in the next PR.

Reply:
Ack!